src/platform/user/profile/vap-svc/components/ProfileInformationEditView.jsx

```diff
@@ -87,7 +87,7 @@ export class ProfileInformationEditView extends Component {
     ) {
       this.interval = window.setInterval(
         this.refreshTransaction,
-        window.VetsGov.pollTimeout || 1000,
+        window.VetsGov.pollTimeout || 2000,
```

Comment:
By setting this after the `||` condition, does that mean that `window.VetsGov.pollTimeout` is never actually set? 🤔